internal/lm/nvml.go

```diff
@@ -218,6 +232,41 @@ func newGPUModeLabeler(devices []resource.Device) (Labeler, error) {
 	return labels, nil
 }
 
+func newImexDomainLabeler(configFile string) (Labeler, error) {
```

Comment:
This diff seems to have removed the clusterUUID and cliqueID labeling (which I think we still want).

Reply:
done, reinstated